src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -94,10 +104,66 @@ private static void OnProcessExit(object sender, EventArgs e)
             }
         }
 
+        private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
+        {
+            using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
+            LogLevel msalLogLevel = LogLevelMap[args.LogLevel];
+            if (_logger.IsLoggingEnabled(msalLogLevel))
```

Comment:
In regards to [this comment](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3777#discussion_r1010386713)

This is the best way to do it. If the logger is not enabled, there should be a null logger which will always return null.

Reply:
I meant to say false above. made a correction

Reply:
Done.